File: cfgs/pythoncfgs/SEQUENCES_GIVEN_LENGTH_EVERY_ELEMENT_EQUAL_TWICE_PREVIOUS.py
Nodes
1;conditional;foo_1 < foo_2;8;
2;conditional;foo_1 == 0;10;
3;statement;foo_1 = fun_1(foo_2 - 1, foo_3) + fun_2(foo_4 // 2, foo_5 - 1);12;
4;statement;return foo_1;13;
5;statement;return 1;11;
6;statement;return 0;9;
7;exit;;
Edges
0;2;6;
1;3;5;
2;4;
3;7;
4;7;
5;7;
6;

File: cfgs/java_cfgs_foo/SEQUENCES_GIVEN_LENGTH_EVERY_ELEMENT_EQUAL_TWICE_PREVIOUS.java
Nodes
1;special;;;
2;conditional;foo_1 < foo_2;14;
3;statement;return 0;14;
4;exit;;;
5;conditional;foo_1 == 0;15;
6;statement;return 1;15;
7;statement;return fun_1(foo_1 - 1, foo_2) + fun_2(foo_3 / 2, foo_4 - 1);16;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;7;
5;4;
6;4;
